src/main/java/org/apache/maven/plugins/assembly/AssemblerConfigurationSource.java

```diff
@@ -223,4 +223,12 @@ public interface AssemblerConfigurationSource {
      * @return Override group name.
      */
     String getOverrideGroupName();
+
+    /**
+     * @return Mask which is applied to permissions of files/directories before they are put into assembly.
```

Comment:
nit: Mask --> mask

Reply:
Sorry, do you mean changing of case? I see all over methods with `@return` Javadoc start from upper case after `@return` statement. Is it OK to not follow existing formatting in this case?

Reply:
Yes, change the case per Javadoc conventions as published by Oracle. Always follow the documented rules. Do not copy what some random developer did 15 years ago. "The description begins with a lowercase letter if it is a phrase (contains no verb), or an uppercase letter if it is a sentence. End the phrase with a period only if another phrase or sentence follows it."

Reply:
Fixed